packages/devtools_app/web/index.html

```diff
@@ -12,7 +12,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
 
-    <title>Dart DevTools</title>
+    <title></title>
```

Comment:
DBC: Should we pre-populate w/ something neutral like 'DevTools'?

Reply:
@kenzieschmoll any thoughts on this? I left this blank to minimize flicker. Now you just go from the URL to Flutter DevTools once the app loads.
If we pre-populate we will go from 
URL -> DevTools -> Flutter DevTools
If we decide to display the name as
`DevTools • Flutter` then we should definitely pre-populate.

Reply:
I am in favor of minimizing the flicker. May be a little overkill, but also could be simple with Flutter animations - what if we added a fade out animation on "DevTools" when connecting to an app. That would make the change look more intentional.

Reply:
There isn't really that much noticeable flicker. Here are a couple demos where we show a generic connect message and avoid displaying an app title until the connection is established. We don't show tabs until the connection is established so this is generally consistent.
![Large GIF (1034x556)](https://user-images.githubusercontent.com/1226812/103942065-a96a1780-50e4-11eb-86b3-4fdfc3725b1b.gif)

![open_connected](https://user-images.githubusercontent.com/1226812/103942550-6fe5dc00-50e5-11eb-9f8f-c563b8284fe0.gif)

Reply:
LGTM this looks good